release/pipeline/configs/environment_config.py

```diff
@@ -42,6 +42,10 @@ def GetDefaultAirflowConfig(branch, gcs_path, mfest_commit, pipeline_type,
   config['PIPELINE_TYPE']      = pipeline_type
   config['VERIFY_CONSISTENCY'] = verify_consistency
   config['VERSION']            = version
+  # MFEST_COMMIT was used for green build, we are transitioning away from it
+  # COMMIT is being used for istio/istio commit sha
+  config['COMMIT']             = commit
```

Comment:
shall we name it as COMMIT_SHA to be more clear?

Reply:
it can also be branch name, _SHA wont fit, will update comment though